### DISCUSSION TITLE
【求助】准大，配置，可否？显示屏，求推荐！

### DISCUSSION THREAD

**Discussion by guiguihha** [share121/inter-knot#1772] [其他]
![工作簿1](https://github.com/user-attachments/assets/d36309a6-de50-4c15-9370-4b837ba67c6e)
老实人配置，应该不会出错吧
有没有1000以下的2k高刷啊

**Comment by tmtzsj** [comment]
你应该问“1000以下就没有好的2k高刷屏！”

**Comment by cpu644** [comment]
整体没有问题 四千124+6750gre确实经典搭配了 不过2k高刷拿675gre可能有些游戏跑不动。一千以下2k高刷屏幕素质到不了顶尖 不过确实能上2k144或者165 只能说搞个aoc去

**Comment by Rain-YYC** [comment]
随便搞一个国产2k180就行了，估计就700~900元子。可以去b站搜一搜攻略
